### ISSUE TITLE
Character mapping or word mapping does not work for me

### ISSUE THREAD

**Issue by PakanAngel** [codelibs/fess#1524]
I want to activate word/character mapping for my searches. 
While indexing documents some characters should changed and mapped to new characters but this is not working. 
How should I activate the mapping?

**Comment by marevol** [comment]
1. Edit mappings in [Mapping List](https://fess.codelibs.org/12.1/admin/mapping-guide.html)
2. Close fess.YYYYMMDD index in [Dashboard](https://fess.codelibs.org/12.1/admin/dashboard-guide.html)
3. Open fess.YYYYMMDD index in [Dashboard](https://fess.codelibs.org/12.1/admin/dashboard-guide.html)
4. Run reindexing with Update Aliases in [Upgrade](https://fess.codelibs.org/12.1/admin/upgrade-guide.html)

**Comment by PakanAngel** [comment]
After closing the main index now when I select reopen index it warns me with an error.

**Comment by PakanAngel** [comment]
I have added to character mappings ي and ك for Persian language.
Then closed the in index and tried to reopen it but I get this error 
{
  "error": {
    "root_cause": [
      {
        "type": "exception",
        "reason": "Failed to verify index [fess.20180218/Y7cGotOnRjOOEMhZGKLsvg]"
      }
    ],
    "type": "exception",
    "reason": "Failed to verify index [fess.20180218/Y7cGotOnRjOOEMhZGKLsvg]",
    "caused_by": {
      "type": "illegal_argument_exception",
      "reason": "match \"ي\" was already added"
    }
  },
  "status": 500

Then I deleted the mapping I added and tried again but unfortunately it won't open the index. 
What should I do be able to open the index again?

**Comment by PakanAngel** [comment]
I am not able to follow this instruction because I tried but failed.
I have added 2 character mappings for ي and ك. These are Arabic characters but wrongly used in many Persian documents and websites.
After closing the index I tried to reopen it but a warning pupped up to my face indicating that these characters were already added. 
Then I removed them and tried several times to reopen the index and finally succeeded. 
But the main question is if they were already added by default why it doesn't work. 
The proof that is doesn't work is when I search ي get different results as I search ی and similar for ك and ک.
